profiling.md

```diff
@@ -121,6 +92,88 @@ The following features are NOT currently supported:
   * Critical path analysis
   * Task dependencies
 
+### Legacy Profile
```

Comment:
Let's move this to the bottom of this section, since we want to encourage people to use the new modes.

Could name it: `Legacy Profile Viewer`

Reply:
Should we update the default mode of the profiler? That was the reason I put this first because I assumed most people would run the new legion_prof like the old one by default, without passing any flags to it. I'm not really sure which mode makes sense to be the new default though? Maybe archive?

Reply:
Yeah, this is something we should think about. I'm not sure what the right answer is. I've been trying to avoid breaking people's workflows, so I haven't tried to change the interface so far.

One thing I thought about was switching to subcommands, so instead of `legion_prof --view` you write `legion_prof view`. And if we're going to make a change like that, we could give a name to the legacy mode while we're at it (e.g., `legion_prof old_archive` or `legion_prof v1_archive` or whatever).

If we have a default mode, I think `view` makes the most sense? Usually you just want to see the logs as easily as possible, and `--view` is the one-step process to do that. Everything else requires multiple steps and some decision on where to put things.